From 4438b59d54589835a0a9d0561ba44c8b3a966209 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Fri, 9 Jan 2026 09:35:36 +0100
Subject: [PATCH 102/N] getcompiler: include GCC in sys.version even for
 clang

some code checks for it, so let's include it for again, but less
prominent.
---
 Python/getcompiler.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Python/getcompiler.c b/Python/getcompiler.c
index 477fe9d..5a9cf76 100644
--- a/Python/getcompiler.c
+++ b/Python/getcompiler.c
@@ -38,9 +38,10 @@
 
 #if defined(__clang__)
 #  if defined(_UCRT)
-#    define COMPILER COMP_SEP "[" ARCH_PREFIX "Clang UCRT " __clang_version__ ARCH_SUFFIX "]"
+// Some code checks for "GCC" in the compiler string, so we include it here.
+#    define COMPILER COMP_SEP "[" ARCH_PREFIX "Clang UCRT " __clang_version__ ARCH_SUFFIX " (GCC)]"
 #  else
-#    define COMPILER COMP_SEP "[" ARCH_PREFIX "Clang " __clang_version__ ARCH_SUFFIX "]"
+#    define COMPILER COMP_SEP "[" ARCH_PREFIX "Clang " __clang_version__ ARCH_SUFFIX " (GCC)]"
 #  endif
 #else
 #  if defined(_UCRT)
